### PR TITLE
Project 1: bubble sort

### PR DESCRIPTION
**Pull request by LOctavio**
In this project, we used 2 methods to sort 2 different arrays, one with numbers and the other with text
we used a block and yield inside the second method.

### CODE REVIEW

**Comment by LOctavio** [comment]
We have a problem with rubocop and we don't know how to fix that problem without editing the .rubocop.yml file.
Screenshot from 2020-04-14 10-03-12

File: .rubocop.yml [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  Enabled: false
+Style/DefWithParentheses:
+  Enabled: false
+Style/FrozenStringLiteralComment:
+  EnforcedStyle: never
+
+Layout/HashAlignment:
+  EnforcedColonStyle: key
+Layout/ExtraSpacing:
+  AllowForAlignment: false
```
Comment (mosaaleb) [inline .rubocop.yml:38]: - [ ] Regarding the linter error: just add this to the end of rubocop file:

```yml
Style/HashEachMethods:
  Enabled: true

Style/HashTransformKeys:
  Enabled: true

Style/HashTransformValues:
  Enabled: true
```

Read more about this issue [here](https://docs.rubocop.org/en/latest/versioning/)

File: bubble_sort.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+  until swap
+    swap = true
+    (0...(arr.length - 1)).each do |i|
+      if yield(arr[i], arr[i + 1]).positive?
+        arr[i], arr[i + 1] = arr[i + 1], arr[i]
+        swap = false
+      end
+    end
+  end
+  puts arr
```
Comment (mosaaleb) [inline bubble_sort.rb]: - [ ] Our method should return the result, the return value of `puts` is nil. We need to return the actual array

**Review by mosaaleb — CHANGES_REQUESTED** [review]
#### Well done @LOctavio @muhenge :clap:

Most requirements are handled, just two minor issues to fix before moving on. Please handle the comments below and submit a new code reivew.


##### Muhammad Ebeid

**Review by mosesogwo — APPROVED** [review]
**Good work, @LOctavio, @muhenge:+1:. Project is approved.**

You can now merge your pull request.

**Cheers:stars:**
### Status: Approved :heavy_check_mark:

**Pull request merged**